docs/integrations/microsoft-azure/azure-functions.md

```diff
@@ -30,33 +41,276 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Operations.png')} alt="Azure Functions Operations dashboard" style={{border: '1px solid gray'}} width="800" />
+
+### OS Statistics
+
+The **Azure Functions - OS Statistics** dashboard provides information on operating system metrics and statistics relevant to your Azure Functions. This includes details on current assemblies, number of processor threads and other OS-level performance indicators.
+
+Use this dashboard to:
+* Track OS-level performance metrics impacting your Azure Functions.
+* Monitor Current assemblies, threads, and handles for optimal performance.
+* Identify and address OS-related performance issues.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-OS-Statistics.png')} alt="Azure Functions OS Statistics dashboard" style={{border: '1px solid gray'}} width="800" />
+
+### Performance
+
+The **Azure Functions - Performance** dashboard provides insights into the performance of your Azure Functions. This includes metrics on execution times, throughput, and overall function efficiency.
+
+Use this dashboard to:
+* Monitor and analyze the performance of your Azure Functions.
+* Identify performance bottlenecks and optimize function execution.
+* Review performance metrics and trends to ensure optimal function performance.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Performance.png')} alt="Azure Functions Performance dashboard" style={{border: '1px solid gray'}} width="800" />
+
+### Health
+
+The **Azure Functions - Health** dashboard provides information of any service health incidents or resource health events associated with Azure Functions in your azure account.
+
+Use this dashboard to:
+* View recent resource and service health incidents.
+* View distribution of service and resource health by incident type.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Health.png')} alt="Azure Functions health dashboard" style={{border: '1px solid gray'}} width="800" />
+
+### Policy and Recommendations
+
+The **Azure Functions - Policy and Recommendations** dashboard provides information of all effect action operations performed by Azure policy and recommendations events from Azure Advisor.
+
+Use this dashboard to:
+* Monitor policy events with warnings and errors.
+* View recent failed policy events.
+* View total recommendation events.
+* Identify High Impact recommendations.
+* View recent recommendation events and navigate to the affected resource.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-Functions/Azure-Functions-Policy-and-Recommendations.png')} alt="Azure Functions - Policy and Recommendations dashboard" style={{border: '1px solid gray'}} width="800" />
```

Comment:
Please export the PNG file of the dashboard and add it to S3 bucket. The image added here looks like a normal screenshot, which actually got widened and the words are not clear in the preview page.

Reply:
Done